pyleecan/GUI/Tools/WTableData/DTableData.py

```diff
@@ -352,6 +352,40 @@ def s_plot(self):
             axes.set_xlabel(self.col_header[0])
             axes.set_ylabel(self.col_header[1])
             fig.show()
+
+        elif (
```

Comment:
Hide DTableData plot button, add plot button in DMatSetup, plot method to move in DMatSetup